package.json

```
@@ -29,12 +29,14 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test ",
+    "precommit": "react-scripts test --watchAll=false && npm run lint ",
```

```suggestion
    "precommit": "react-scripts test --watchAll=false && npm run check && npm run lint",
```
It will be better to add a formatting check in the pre-commit hook.